ProjectVoyage/Cell/ChatCell.m

```diff
@@ -8,27 +8,21 @@
 #import "ChatCell.h"
 #import "Util.h"
 
-@implementation ChatCell
-
-- (void)awakeFromNib {
-    [super awakeFromNib];
-    // Initialization code
-}
-
-
-- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
-    [super setSelected:selected animated:animated];
-    // Configure the view for the selected state
-}
```

Comment:
We don't want to commit code unless there is logic here that needs to be overriden